documentation/validation_log.rst

```diff
@@ -59,6 +59,8 @@ New Features:
     * updated to allow the mask and summed power arrays to be configured, either directly in the configuration file, or with a path to another file (such as that output by the above)
         * tested in file value arrays by setting in a file and calling write mask to ensure the values are in the output file
         * tested  from-file by modifying the above output file (so that the values differ), configuring with it as input, and the writing a new output to compare
+    * added support for specifying thresholds to be measured in units of sigma of the noise, in addition to power (in dB, amplitude, etc.)
+        * building a mask now must accumulate variance data as well as power data
```

Comment:
@cclaessens please add notes about the tests which were completed (and fix my notes on these lines if needed)